### PR TITLE
Skill - Guts

### PR DESCRIPTION
**Pull request by JesterWizard**
+5 attack while afflicted by any status condition.

### CODE REVIEW

File: Wizardry/Core/UnitStatusGetter/source/PowGetter.c [hunk 1]
```diff
@@ -74,5 +74,10 @@ int PowGetterSkills(int status, struct Unit * unit)
         status += 7;
 #endif
 
+#if defined(SID_Guts) && (SID_Guts < MAX_SKILL_NUM)
+    if (SkillTester(unit, SID_Guts) && (unit->statusIndex != UNIT_STATUS_NONE))
```
[the remark this answers is not in the document]
Comment (JesterWizard) [reply Wizardry/Core/UnitStatusGetter/source/PowGetter.c]: Changed to use the GetUnitStatusIndex() function instead.

**Comment by MokhaLeee** [comment]
WRONG calculation. PowGetter is just used in case of non-magic attack. This PR may just cause +5 pow

**Comment by JesterWizard** [comment]
> WRONG calculation. PowGetter is just used in case of non-magic attack. This PR may just cause +5 pow

Would it be better to apply it to the MagGetter too then, so both stats are boosted, or do you have another suggestion?

**Comment by MokhaLeee** [comment]
I think throw them to pre-battle calc will be better

**Comment by JesterWizard** [comment]
I'm gonna close and reopen a new PR for this, since there's been some significant changes to the prebattle.c file since.

**Pull request closed without merging**